File: make_icons.js
const sharp = require('sharp');
const path = require('path');
const fs = require('fs');

const base = path.join(__dirname, '..');
const input = path.join(base, 'bgr (1).jpg');
const outDir = path.join(base, 'icons');
if (!fs.existsSync(outDir)) fs.mkdirSync(outDir, { recursive: true });

const sizes = [72, 192, 512];

async function make() {
  for (const s of sizes) {
    const out = path.join(outDir, `icon-${s}x${s}.png`);
    await sharp(input).resize(s, s).png({ quality: 90 }).toFile(out);
    console.log('Wrote', out);
  }
}

make().catch(err => { console.error(err); process.exit(1); });

File: make_icons.py
from PIL import Image
import os

# Input image (replace path if needed)
input_path = os.path.join(os.path.dirname(__file__), '..', 'bgr (1).jpg')
output_dir = os.path.join(os.path.dirname(__file__), '..', 'icons')

sizes = [72, 192, 512]

os.makedirs(output_dir, exist_ok=True)

with Image.open(input_path) as im:
    im = im.convert('RGBA')
    for s in sizes:
        out = im.copy()
        out = out.resize((s, s), Image.LANCZOS)
        out_path = os.path.join(output_dir, f'icon-{s}x{s}.png')
        out.save(out_path, format='PNG', optimize=True)
        print('Wrote', out_path)

print('Done')
